webrtc/videoroom/assets/src/membraneWebRTC.ts

```diff
@@ -34,9 +25,16 @@ export class MembraneWebRTC {
   private _channel?: Channel;
   private channelId: string;
 
-  private localStream: MediaStream;
+  private localTracks: Set<MediaStreamTrack> = new Set<MediaStreamTrack>();
   private remoteStreams: Set<MediaStream> = new Set<MediaStream>();
   private connection?: RTCPeerConnection;
+  private rtc_config: RTCConfiguration = {
```

Comment:
If we don't modify the config it could be `readonly`.

Reply:
We modify it in the constructor, is that still readonly?

Reply:
From what I learned in typescript playground you can do that. It will be readonly outside of construtor.